iset.mm

```diff
@@ -148169,6 +148242,320 @@ mathematician Ptolemy (Claudius Ptolemaeus).  This particular version is
+
+  $( The logarithm of the quotient of two positive real numbers is the
+     difference of logarithms.  Property 3 of [Cohen4] p. 361.  (Contributed by
+     AV, 29-May-2020.) $)
+  rprelogbdiv $p |- ( ( ( B e. RR+ /\ B =//= 1 ) /\ ( A e. RR+ /\ C e. RR+ ) )
+                 -> ( B logb ( A / C ) ) = ( ( B logb A ) - ( B logb C ) ) ) $=
+    ( crp wcel c1 cap wa cneg co cmul clogb caddc cdiv cr wceq cc adantl oveq2d
+    wbr ccxp cmin neg1rr rprelogbmulexp mp3anr3 rpcn adantr cc0 rpap0 divrecapd
+    ax-1cn rpcxpneg mpan2 rpcxp1 eqtrd eqtr4d simpll simplr simprr syl3anc recn
+    rplogbcl mulm1 3syl simprl recnd negsubd eqtr2d 3eqtr4d ) BDEZBFGTZHZADEZCD
+    EZHZHZBACFIZUAJZKJZLJZBALJZVQBCLJZKJZMJZBACNJZLJWAWBUBJZVLVMVNVQOEVTWDPUCAB
+    CVQUDUEVPWEVSBLVOWEVSPVLVOWEAFCNJZKJVSVOACVMAQEVNAUFUGVNCQEVMCUFRVNCUHGTVMC
+    UIRUJVOVRWGAKVNVRWGPVMVNVRFCFUAJZNJZWGVNFQEVRWIPUKCFULUMVNWHCFNCUNSUORSUPRS
+    VPWDWAWBIZMJZWFVPWBOEZWBQEZWDWKPVPVJVKVNWLVJVKVOUQZVJVKVOURZVLVMVNUSBCVBUTZ
+    WBVAWMWCWJWAMWBVCSVDVPWAWBVPWAVPVJVKVMWAOEWNWOVLVMVNVEBAVBUTVFVPWBWPVFVGVHV
+    I $.
+
+  $( Identity law for general logarithm: the logarithm of a power to the base
+     is the exponent.  Property 6 of [Cohen4] p. 361.  (Contributed by Stefan
+     O'Rear, 19-Sep-2014.)  (Revised by AV, 9-Jun-2020.) $)
+  relogbexpap $p |- ( ( B e. RR+ /\ B =//= 1 /\ M e. ZZ )
+                    -> ( B logb ( B ^ M ) ) = M ) $=
+    ( crp wcel c1 cap wbr cz w3a cexp co clogb cmul wceq simp1 simp2 rplogbzexp
+    simp3 syl211anc rplogbid1 3adant3 oveq2d zcnd mulid1d 3eqtrd ) ACDZAEFGZBHD
+    ZIZAABJKLKZBAALKZMKZBEMKBUIUFUGUFUHUJULNUFUGUHOZUFUGUHPUMUFUGUHRZAABQSUIUKE
+    BMUFUGUKENUHATUAUBUIBUIBUNUCUDUE $.
+
+  $( Identity law for general logarithm with integer base.  (Contributed by
+     Stefan O'Rear, 19-Sep-2014.)  (Revised by Thierry Arnoux, 27-Sep-2017.) $)
+  nnlogbexp $p |- ( ( B e. ( ZZ>= ` 2 ) /\ M e. ZZ ) ->
+    ( B logb ( B ^ M ) ) = M ) $=
+    ( c2 cfv wcel cz wa cc0 wceq cexp co clogb wne c1 crp syl2anc ad2antrr clog
+    adantr cdiv cuz cap wbr w3a zgt1rpn0n1 simp1d simp3d wb eluzelz 1zzd mpbird
+    zapne rplogb1 simpr oveq2d cc rpcnd exp0d 3eqtr4d simplr rpexpcld rplogbval
+    eqtrd cmul syl3anc ccxp zred logcxpd cn cxpexpnn fveq2d eqtr3d oveq1d recnd
+    eluz2nn relogcld logrpap0d divcanap4d 3eqtr2d wo wn wdc zdceq mpan2 exmiddc
+    0z syl df-ne orbi2i sylibr adantl mpjaodan ) ACUADEZBFEZGZBHIZAABJKZLKZBIBH
+    MZWOWPGZANLKZHWRBWTAOEZANUBUCZXAHIWOXBWPWOXBAHMZANMZWMXBXDXEUDWNAUEZSUFZSWM
+    XCWNWPWMXCXEWMXBXDXEXFUGWMAFENFEXCXEUHCAUIWMUJANULPUKZQAUMPWTWQNALWTWQAHJKN
+    WTBHAJWOWPUNZUOWTAWOAUPEWPWOAXGUQSURVCUOXIUSWOWSGZWRWQRDZARDZTKZBXLVDKZXLTK
+    BXJXBXCWQOEWRXMIWOXBWSXGSZWMXCWNWSXHQZXJABXOWMWNWSUTZVAAWQVBVEXJXNXKXLTXJAB
+    VFKZRDXNXKXJABXOXJBXQVGZVHXJXRWQRXJAVIEZWNXRWQIWMXTWNWSAVOQXQABVJPVKVLVMXJB
+    XLXJBXSVNXJXLXJAXOVPVNXJAXOXPVQVRVSWNWPWSVTZWMWNWPWPWAZVTZYAWNWPWBZYCWNHFEY
+    DWFBHWCWDWPWEWGWSYBWPBHWHWIWJWKWL $.
```

Comment:
I think this proof can be shortened using ~relogbexpap .

Reply:
Done